db/migrate/20161211162552_add_generating_ems_to_event_streams.rb

```diff
@@ -0,0 +1,5 @@
+class AddGeneratingEmsToEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :generating_ems_id, :bigint, :after => :ems_id
```

Comment:
also, I guess you'll want to index this field.
If you use `references` it'll add the index automagically

http://guides.rubyonrails.org/active_record_migrations.html